resolwe_bio/tests/processes/test_pca.py

```diff
@@ -4,6 +4,78 @@
 
 
 class PcaProcessorTestCase(KBBioProcessTestCase):
+    @with_resolwe_host
+    @tag_process("pca-beta")
+    def test_beta_pca(self):
+        with self.preparation_stage():
+            expression_1 = self.prepare_expression(
```

Comment:
One test case could be uploaded tpms with no rc and then running the PCA (`low_expression_filter` turned on). It should fail since no raw counts are available.